docs/deployment/0-crd.yaml

```diff
@@ -11,3 +11,147 @@ spec:
     singular: bundle
   scope: Namespaced
   version: v1
+  validation:
```

Comment:
why do we have 0-crd.yaml when Smith creates it automatically?

Reply:
It does not anymore https://github.com/atlassian/smith/pull/184